src/xcode/ENA/ENA/Source/Models/Metadata/Key Submission Metadata/KeySubmissionMetadata.swift

```diff
@@ -28,8 +31,9 @@ struct KeySubmissionMetadata: Codable {
 		self.advancedConsentGiven = advancedConsentGiven
 		self.hoursSinceTestResult = hoursSinceTestResult
 		self.hoursSinceTestRegistration = hoursSinceTestRegistration
-		self.daysSinceMostRecentDateAtRiskLevelAtTestRegistration = daysSinceMostRecentDateAtRiskLevelAtTestRegistration
-		self.hoursSinceHighRiskWarningAtTestRegistration = hoursSinceHighRiskWarningAtTestRegistration
+		self.daysSinceMostRecentDateAtENFRiskLevelAtTestRegistration = daysSinceMostRecentDateAtENFRiskLevelAtTestRegistration
```

Comment:
Missing the assignment for daysSinceMostRecentDateAtCheckinRiskLevelAtTestRegistration and hoursSinceCheckinHighRiskWarningAtTestRegistration

Reply:
yep. added.